src/Illuminate/Http/Client/PendingRequest.php

```diff
@@ -439,6 +447,19 @@ public function withUserAgent($userAgent)
         });
     }
 
+    /**
+     * Specify the URL parameters that can be substituted into the request URL.
+     *
+     * @param  array  $parameters
+     * @return $this
+     */
+    public function withUrlParameters(array $parameters = [])
+    {
+        return tap($this, function () use ($parameters) {
+            $this->urlParameters = $parameters;
+        });
```

Comment:
Is it the same than writing

```suggestion
        $this->urlParameters = $parameters;

        return $this;
```
Or am I missing something?